projects/Mallard/src/hooks/use-edition-provider.tsx

```diff
@@ -44,6 +49,14 @@ export interface StoreSelectedEditionFunc {
     ): void
 }
 
+export const getSpecialEditionProps = (
+    edition: RegionalEdition | SpecialEdition,
```

Comment:
Dont see why we would ever have the RegionalEdition in this case?

Reply:
the point of the function is that we don't know  whether `edition` is a special edition or a regional edition

Reply:
As with the previous comment, it makes sense why this now has to exist in the way it does.